src/cpu/x64/jit_avx512_common_convolution.cpp

```diff
@@ -267,24 +270,25 @@ void jit_avx512_common_convolution_fwd_t<src_type, wei_type,
                         ic_work = utils::this_block_size(icb * jcp.ic_block,
                                 jcp.ic, icb_step * jcp.ic_block);
                     }
+                    int oc_off = oc_off_idx * (is_dst_layout_nxc ? 1 : jcp.oc_block) * sizeof(dst_data_t);
```

Comment:
Good catch! In fact this convolution is always working with fp32 data type on inputs and outputs so everything works correct.  Anyway I changed to "float" in order to avoid painful debug in the future.